cmd/core/main.go

```diff
@@ -216,7 +216,7 @@ func waitWebhookSecretVolume(certDir string, timeout, interval time.Duration) er
 				// nolint
 				defer f.Close()
 				// check if dir is empty
-				if _, err := f.Readdir(1); err == io.EOF {
+				if _, err := f.Readdir(1); errors.Is(err, io.EOF) {
```

Comment:
Just curious, is using function better?

Reply:
As the linter says, `comparing with == will fail on wrapped errors. Use errors.Is to check for a specific error`